src/Parser/AfflictionWarlock/Modules/WarlockCore/DeathsEmbrace.js

```diff
@@ -54,19 +49,6 @@ class DeathsEmbrace extends Module {
     }
     this.bonusDmg += getDamageBonus(event, this.getDeathEmbraceBonus(targetHealthPercentage));
   }
-
-  statistic() {
```

Comment:
I reworked this module in next commits. This mechanic is used by both the talent module and Soul of the Netherlord. I will be doing another PR this evening and this exact module is moved back into talents, and 2 new modules are created - DeathsEmbraceTalent and SoulOfTheNetherlord that both just get this module as a dependency and display the bonusDmg, but this time in its correct place (StatisticBox if it's from a talent, Item if it's from the legendary finger).